ABObjectApiCore.js

```diff
@@ -65,4 +65,62 @@ module.exports = class ABObjectApiCore extends ABObject {
 
       return model;
    }
+
+   /**
+    * @function getPagingValues()
+    *
+    * @return {Object} - {
+    *                       start: "Property name of the API for start index",
+    *                       limit: "Property name of the API for limit return the item number"
+    *                     }
+    */
+   getPagingValues({ skip, limit }) {
+      const result = {};
+      const pagingSettings = this.request?.paging ?? {};
+
+      if (pagingSettings.start && skip != null) {
+         result[pagingSettings.start] = skip;
+      }
+      if (pagingSettings.limit && limit != null) {
+         result[pagingSettings.limit] = limit;
+      }
+
+      return result;
+   }
+
+   dataFromKey(data) {
+      let result = [];
+
+      if (!Array.isArray(data)) data = [data];
+
+      data.forEach((item) => {
+         // Clone item
+         let itemResult = { ...item };
+
+         (this.response.dataKey ?? "").split(".").forEach((key) => {
```

Comment:
This took me a couple minutes to understand. Can you add a comment explaining how we use the data key to find the relevant result?